lib/DDG/Spice/Maps/Maps.pm

```diff
@@ -4,9 +4,10 @@ package DDG::Spice::Maps::Maps;
 use strict;
 use DDG::Spice;
 
-spice to => 'http://open.mapquestapi.com/nominatim/v1/search?format=json&json_callback={{callback}}&addressdetails=1&polygon=1&bounded=0&q=$1';
+spice to => 'http://api.mapbox.com/v4/geocode/mapbox.places/$1.json?access_token={{ENV{DDG_SPICE_MAPBOX_KEY}}}&callback={{callback}}';
 spice is_cached => 0;
 spice proxy_cache_valid => "418 1d";
+spice wrap_jsonp_callback => 1; 
```

Comment:
This isn't necessary if the API has `callback` param. It's only for when the API doesn't let you specify `&callback={{callback}}` so we can tell NGINX to wrap the JSON response.